src/database/sql_database.cc

```diff
@@ -988,7 +988,7 @@ std::vector<std::shared_ptr<CdsObject>> SQLDatabase::search(std::unique_ptr<Sear
 
     // order by code..
     auto orderByCode = [&]() {
-        SortParser sortParser(searchColumnMapper, param->getSortCriteria());
+        SortParser sortParser(searchColumnMapper, param.getSortCriteria());
```

Comment:
This can use auto

Reply:
I didn't want to go too far with one commit, but I have seen things that I don't like either. Maybe I'll spend a future PR on this. Currently, this will have no functional effect/improvement.